nmdc_server/migrations/versions/09a705b0bc00_submission_images.py

```diff
@@ -0,0 +1,103 @@
+"""Add submission_images_object table.
+
+The name of the table indicates that stores references to objects in the nmdc-submission-images
```

Comment:
I think a word is missing.

```suggestion
The name of the table indicates that it stores references to objects in the nmdc-submission-images
```